docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
     depends_on: [ signal ]
 
   signal:
-    image: bbernhard/signal-cli-rest-api:0.81
+    image: bbernhard/signal-cli-rest-api:0.162-dev
```

Comment:
There is an issue with sending messages by uuid, which has been fixed, but not yet released. See https://github.com/bbernhard/signal-cli-rest-api/issues/550